Directory.Build.props

```diff
@@ -2,22 +2,30 @@
 <Project>
 
   <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <RepositoryUrl>https://github.com/dotnet/sdk</RepositoryUrl>
-    <PackageProjectUrl>$(RepositoryUrl)</PackageProjectUrl>
-
     <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
     <LangVersion>Latest</LangVersion>
   </PropertyGroup>
 
-  <Import Project="Sdk.props" Sdk="RoslynTools.RepoToolset" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
     <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <DebugType>embedded</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+
+    <!--
+      Tools and packages produced by this repository support infrastructure and are not shipping on NuGet or via any other official channel.
+    -->
+    <IsShipping>true</IsShipping>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- 
+      'NetFxTfm' is the standard desktop Target Framework Moniker which this repo's packages are targeting
+      ie.  Place 'NetFxTfm' in the 'TargetFramework' property of a csproj like <TargetFrameworks>$(NetFxTfm);netcoreapp2.0</TargetFrameworks> 
+    -->
+    <NetFxTfm>net461</NetFxTfm>
```

Comment:
This is used by arcade somehow?

Reply:
I'm not seeing it in the diff anywhere else. Comment suggests we should be putting this in the project files.

Reply:
I will follow up with Chris on this.